scraper/scraper_doctolib.py

```diff
@@ -0,0 +1,108 @@
+import httpx
+import requests
+import os
+import re
+from bs4 import BeautifulSoup
+import csv
+import json
+import time
+
+insee = {}
+
+
+def search(max_page=1000):
+
+    BASE_URL = "https://www.doctolib.fr"
+    RECHERCHE_URL = "/vaccination-covid-19/france?"
+    PARAMETRES = "ref_visit_motive_ids[]=6970&ref_visit_motive_ids[]=7005&ref_visit_motive_ids[]=7107"
+
+    DOCTOLIB_SLOT_LIMIT = 50
+
+    DOCTOLIB_HEADERS = {
```

Comment:
Ces "variables" sont plutôt des constantes et devraient être placées après les import, out of scope des fonctions

Reply:
Code mis en commun avec doctolib.py